pages/common/cowsay.md

```diff
@@ -1,24 +1,28 @@
 # cowsay
 
-> Generate an ASCII character (by default a cow) saying or thinking something.
+> Print an ASCII friend (by default a cow) saying or thinking something.
 > More information: <https://github.com/tnalpgge/rank-amateur-cowsay>.
 
-- Print an ASCII cow saying "Hello world":
+- Print an ASCII cow saying "Hello World":
 
-`cowsay "Hello world"`
+`cowsay "{{Hello World}}"`
 
 - Read text from stdin for the balloon:
 
-`echo "Hello" | cowsay`
+`echo "{{Hello World}}" | cowsay`
 
-- List all available characters:
+- List all available friends:
```

Comment:
I like to use "valid" for list commands like this, because it clarifies it a bit more than "available" 🤔
```suggestion
- List all valid friends:
```

Reply:
I feel valid implies that there are situations when you cant use the friends; available does as well but less so

Reply:
Maybe you're right — let's skip this.